configure

```diff
@@ -78,10 +78,21 @@ done
 
 
 # Find out where the CUDA toolkit is installed
+OSNAME=`uname -s`
+
 while true; do
   # Configure the Cuda SDK version to use.
   if [ -z "$TF_CUDA_VERSION" ]; then
-    read -p "Please specify the Cuda SDK version you want to use, e.g. 7.0. [Leave empty to use system default]: " TF_CUDA_VERSION
+    if [ "$OSNAME" == "Linux" ]; then
+      default_cuda_version='7.0'
```

Comment:
Isn't the default version on Linux 7.5 and the default version on OS X 7.0 ?  You have it the other way here.